### PR TITLE
Replace cli with manage.py

### PR DESCRIPTION
**Pull request by skygeo**
## Changes
* Replace cli with python mange.py
* Specify app configuration through the APP_CONFIG environment variable
* Group api endpoints using flask blueprints.

### CODE REVIEW

File: manage.py [hunk 1]
```diff
@@ -5,7 +5,11 @@
 connection string of a database to connect to, e.g.:
     $ export SLINGSHOT_DATABASE_URI=sqlite:////tmp/mydb.db
```
Comment (gnhenderson) [inline manage.py]: Everything from line 2 to line 6 is no longer relevant. Line 2 could be something like:
`Script for managing the application and its backend database from the command line`

Lines 4, 5, & 6 can be completely cut if we are no longer using the `SLINGSHOT_DATABASE_URI` variable anywhere and are using the native `SQLALCHEMY_DATABASE_URI` instead.

**Comment by gnhenderson** [comment]
`requirements.txt` still has the following:
```
click==6.6
configobj==5.0.6
...
validator.py==1.2.5
```
If these ^^^ are no longer required can they be removed from the requirements? Otherwise LGTM

File: manage.py (hunk 1 above)
Comment (chaudyg) [inline manage.py]: done

**Comment by chaudyg** [comment]
@gnhenderson I've made the suggested changes

**Comment by gnhenderson** [comment]
LGTM

**Comment by gnhenderson** [comment]
Sorry - didn't mean to close that...

**Pull request merged**